packages/lab/src/combo-box-next/ComboBoxNext.tsx

```diff
@@ -46,35 +40,37 @@ export interface ComboBoxNextProps<T>
    */
   PortalProps?: UseComboBoxPortalProps;
   /**
-   * If `true`, the component will be disabled.
+   * Controlled prop. Controls the value for the Combo Box input.
+   */
+  value?: string;
```

Comment:
`value` as top level prop could be confusing for users?

Reply:
@origami-z would you suggest changing it for inputValue?

Reply:
If it's not commonly used... don't have it at the top level and make it available in `InputProps` or somthing?